src/qt/rpcconsole.cpp

```diff
@@ -523,6 +525,16 @@ void RPCConsole::setNumBlocks(int count, const QDateTime& blockDate)
     ui->lastBlockTime->setText(blockDate.toString());
 }
 
+void RPCConsole::setMempoolSize(int numberOfTxs, size_t dynUsage)
+{
+    ui->mempoolNumberTxs->setText(QString::number(numberOfTxs));
+
+    if (dynUsage < 1024*1024)
+        ui->mempoolSize->setText(QString::number(dynUsage/1024.0, 'f', 2) + " KB");
+    else
+        ui->mempoolSize->setText(QString::number(dynUsage/1024.0/1024.0, 'f', 2) + " MB");
```

Comment:
In bitcoin core we stick to SI units and use MB, which is defined as 1e6, not 1024*1024, MiB, etc if we do that's a mistake.